recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   sha256: b854a6d9049ad9e15458a7288ae719f7fbc842928d952753c9dabf8061350b29
 
 build:
-  number: 0
+  number: 1000
```

Comment:
@ocefpaf @jakirkham any of you know why this was set to ``1000`` ?

Reply:
This is part of the transition, it will be reverted back to `0` when we swap the binaries.